sources/packages/api/src/services/application/application.service.ts

```diff
@@ -572,7 +572,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
       );
     } else {
       applicationQuery.orderBy(
-        `CASE application.applicationStatus
+        `CASE application.application_status
```

Comment:
why `application_status` instead of  `applicationStatus`?

Reply:
There is an unknown issue where apparently the model property is not translated to the column name. I asked @guru-aot to add a todo and do the investigation in the next PR right after this one. Do you see any concern about it @ann-aot?

Reply:
okay...no, I just want to know the reason.